src/content/docs/new-relic-control/agent-control/configuration.mdx

```diff
@@ -0,0 +1,284 @@
+
+              #chart_values:
+              #  newrelic-logging:
+              #    sendMetrics: true
+          agent-operator:
+            type: com.newrelic.k8s_agent_operator:0.1.0
+            chart_version: "*"
+    ```
+  </Collapser>
+
+</CollapserGroup>
+
+
+
+### Local configuration: OpenTelemetry
+The following snippet demonstrates installing Agent Control with OpenTelemetry and disabling the filelog receiver in the managed [`nr-k8s-otel-collector` Helm chart](https://github.com/newrelic/helm-charts/tree/master/charts/nr-k8s-otel-collector#values):
+
+<CollapserGroup>
+  <Collapser
+    id="otel-config"
+    title="OpenTelemetry configuration"
+  >
+
+    ```yaml
+        global:
+          cluster: "YOUR_CLUSTER_NAME"
+          licenseKey: "YOUR_LICENSE_KEY"
+        # Values related to the agent control's Helm chart release.
+        # `https://github.com/newrelic/helm-charts/blob/master/charts/agent-control/values.yaml`
+        agent-control-deployment:
+          identityClientId: ""
+          identityClientSecret: ""
+          config:
+            fleet_control:
+              # Optional: Specify a fleet_id (entity guid) to automatically connect to an existing fleet.
+              fleet_id: ""
+              auth:
+                # New Relic organization ID where agent will connect to.
+               organizationId: "YOUR_ORGANIZATION_ID"
+          
+            # List of managed agents that will be deployed. The key represents the name of the agent and the value holds the configuration.
+           subAgents:
+             infrastructure:
+               type: newrelic/com.newrelic.infrastructure:0.1.0
+               content:
+                  # Ref: `https://github.com/newrelic/helm-charts/tree/master/charts/nri-bundle%60 
+                  # Recommended: check and define an explicit chart version (latest stable)
+                 chart_version: "*"
+             agent-operator:
+               type: newrelic/com.newrelic.k8s_agent_operator:0.1.0
+               content:
+                 chart_version: "*"
+             fluentbit:
+               type: newrelic/io.fluentbit:0.1.0
+               content:
+                  # Ref: `https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-logging`
+                  # Recommended: check and define an explicit chart version (latest stable)
+                 chart_version: "*"
+                 chart_values:
+                   global:
+                     lowDataMode: true
+             prometheus:
+               type: newrelic/com.newrelic.prometheus:0.1.0
+               content:
+                 chart_version: "*"
+                 chart_values:
+                   global:
+                     lowDataMode: true
+                   newrelic-prometheus-agent:
+                     config.kubernetes.integrations_filter.enabled: false
+    ```
+
+  </Collapser>
+
+</CollapserGroup>
+
+### Remote configuration: New Relic infrastructure
```

Comment:
I think it is not clear that locally we had a configuration for AC + 1 for each subagent

In the remove config section you added examples for the subAgents and no AC config is there